.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -9,20 +9,20 @@ body:
   - type: markdown
     attributes:
       value: |
-        Before crreating a feature request, please search for the behaviour in the existing issues. Keep in mind that we use the [agile style](https://www.agilealliance.org/glossary/user-story-template/) for our user stories
+        Before creating a feature request, please search for the behaviour in the existing issues. Keep in mind that we use the [agile style](https://www.agilealliance.org/glossary/user-story-template/) for our user stories
```

Comment:
"Before creating a feature request, double-check the backlog to ensure the requested functionality isn't already documented."

Might be a clearer way of wording this.

Reply:
Did more this time, besides just spelling mistakes. pls review once more